Bibliotheque nationale de France.js

```diff
@@ -506,14 +541,10 @@ var BnfClass = function() {
         var url = record.getFieldSubfields("856");
 		
 		if(url && url.length > 0 && url[0].u) {		
-		 item.attachments = [
-				{
-					url: url[0].u,
-					title: "Bnf Gallica entry", 
-					mimeType: "text/html", 
-					snapshot:false
-				}
-			];
+
+		 item.url = url[0].u;
```

Comment:
just to make sure I understand this right -- Gallica has all full texts available online, right? So the url would link to the actual resource, not just a catalog entry?